views/tracks.hbs

```diff
@@ -0,0 +1,7 @@
+<div>
+{{#each answer1}}
+<p style="color: blueviolet;">{{this.name}}</p>
+<a href="{{this.preview_url}}">Preview Song</a>
```

Comment:
You have an `<audio> tag` that you could've used to propper display the previews of the songs on your webpage.
```<audio controls src='{{this.preview_url}}'></audio>```